.gitignore

```diff
@@ -1,24 +1,34 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
-*.pyc
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
 
-hg-data/histogram_tools.py
+# testing
+/coverage
 
-hg-data/Histograms.json
+# production
+/build
 
-hg-data/nsDeprecatedOperationList.h
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
 
-hg-data/UseCounters.conf
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
 
+# pre-react ignores
```

Comment:
I'm not sure if I understand what this comment means.

Reply:
Old ignores, will remove along with the mercurial ones above.